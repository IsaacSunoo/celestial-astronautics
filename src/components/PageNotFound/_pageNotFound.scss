.page-not-found {
    padding-top: 30px;
    text-align: center;
}