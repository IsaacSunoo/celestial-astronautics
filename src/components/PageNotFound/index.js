import React from 'react';
import { Link } from 'react-router-dom';

const PageNotFound = () => {
    return (
        <div className='page-not-found'>
            <h2>Page Not Found.</h2>
            <h2 className='not-found'>404</h2>
            <Link to='/'>Return Home</Link>
        </div>
    )
}

export default PageNotFound;