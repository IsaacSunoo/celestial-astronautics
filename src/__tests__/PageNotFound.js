import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from '../components/PageNotFound';

describe('Page Not Found', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<PageNotFound />);
    });

    it('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});