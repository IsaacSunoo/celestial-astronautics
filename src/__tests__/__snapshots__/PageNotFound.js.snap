// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Not Found should match snapshot 1`] = `
<div
  className="page-not-found"
>
  <h2>
    Page Not Found.
  </h2>
  <h2
    className="not-found"
  >
    404
  </h2>
  <Link
    to="/"
  >
    Return Home
  </Link>
</div>
`;
